lib/quarry.rb

```diff
@@ -195,6 +199,15 @@ def arch_to_pkg(arch_name)
   return [name, slot]
 end
 
+def get_official_packages
+  if OFFICIAL_PACKAGES.empty?
```

Comment:
Standard ruby naming suggests UPPER_CASE should be used for constant value. I'll change it locally in my git client.

Other than that CL looks great.

Reply:
force pushed a fixed version

Reply:
hmm, that causes:

```
Traceback (most recent call last):
	2: from ./lib/update.rb:20:in `<main>'
	1: from /home/eschwartz/git/archlinux/quarry/lib/quarry.rb:614:in `build_packages'
/home/eschwartz/git/archlinux/quarry/lib/quarry.rb:203:in `get_official_packages': undefined local variable or method `official_packages' for main:Object (NameError)
```

Reply:
Oh, because ruby considers `Variable` to be global, `variable` to be function-local, and `$variable` to be global/exported. Wow...